src/DynamoPackages/PackageLoader.cs

```diff
@@ -258,13 +260,29 @@ private void TryLoadPackageIntoLibrary(Package package)
                             OnRequestLoadNodeLibrary(assem.Assembly);
                             loadedNodeLibs.Add(assem.Assembly);
                         }
-                        catch (LibraryLoadFailedException ex)
+                        catch (Exception ex)
                         {
-                            Log(ex.GetType() + ": " + ex.Message);
+                            failedNodeLibs.Add(assem.Assembly);
+                            if (ex is LibraryLoadFailedException)
+                            {
+                                // Managed exception
+                                // We can still try to load other parts of the package
+                                Log(ex.GetType() + ": " + ex.Message);
+                            }
+                            else
+                            {
+                                // Everything else is considered fatal to the package loading operation.
+                                fatalError = true;
```

Comment:
I guess I am silently discarding `AssemblyBlockedException` exceptions here...
Not sure how to make those fit in this scenario...
Maybe if all assemblies fail....with the same exception type `AssemblyBlockedException`...then we should re throw that ?...

Reply:
Yes, I think we should. Can you please do that?

Reply:
I think it's cleaner if you do:
```
catch (LibraryLoadFailedException ex)
{ 
...
}
catch (Exception ex)
{
...
}
```